.gitignore

```diff
@@ -1,14 +0,0 @@
-tests/chromedriver
-*DS_Store*
-node_modules*
-.env
-npm-debug.log
-error*.jpg
-screenshot_*.jpg
-downloads_*
-*_lang_*.feature
-*_report_*
-_al_test_project_name.json
-.history
-__pycache__/
-docassemble.AssemblyLine.egg-info/*
```

Comment:
Definitely no reason to delete this file. Even if some of the entries in here don't matter any more, other still do (.DS_STORE, __pycache__, etc.)

Reply:
Dang, didn't realize more was added to this. I'll restore this as well and be more selective about the contents 👍

Reply:
IMO you don't need to be selective about the contents, just leave it all there!